homeassistant/components/august.py

```diff
@@ -141,11 +141,11 @@ def setup(hass, config):
     from requests import Session
 
     conf = config[DOMAIN]
+    api_http_session = None
     try:
         api_http_session = Session()
     except RequestException as ex:
         _LOGGER.warning("Creating HTTP session failed with: %s", str(ex))
```

Comment:
What happens with the api in this case? Can it handle `http_session` being `None`?

Reply:
Yes, if it is None then it will use request instead of a session.